### PR TITLE
fix: reload balances, txs and channels after transfer is complete

### LINKED ISSUE #692: Bug: migrate shared funds on wallet page does not refresh the wallet balance and transaction list

### PR DESCRIPTION
**Pull request by im-adithya**
Fixes #692

Also reloading channels so the liquidity changes in Node screen

### CODE REVIEW

File: frontend/src/screens/channels/Channels.tsx [hunk 1]
```diff
@@ -67,9 +67,9 @@ import { request } from "src/utils/request";
 
 export default function Channels() {
   useSyncWallet();
-  const { data: channels } = useChannels();
+  const { data: channels, mutate: reloadChannels } = useChannels();
```
Comment (rolznz) [inline frontend/src/screens/channels/Channels.tsx:70]: why do we need to reload the channels?

File: frontend/src/screens/wallet/index.tsx [hunk 2]
```diff
@@ -61,7 +63,11 @@ function Wallet() {
               <TransferFundsButton
                 channels={channels}
                 albyBalance={albyBalance}
-                reloadAlbyBalance={reloadAlbyBalance}
+                onTransferComplete={async () => {
+                  await reloadAlbyBalance();
```
Comment (rolznz) [inline frontend/src/screens/wallet/index.tsx:67]: You could use `Promise.all` here, and then there is no need to make the function async if it returns a single promise

File: frontend/src/screens/channels/Channels.tsx (hunk 1 above)
Comment (im-adithya) [reply frontend/src/screens/channels/Channels.tsx:70]: The liquidity also changes right?

File: frontend/src/screens/wallet/index.tsx (hunk 2 above)
Comment (im-adithya) [reply frontend/src/screens/wallet/index.tsx:67]: Ah nice, thanks 🙌 I was also thinking how to change as this looks weird

File: frontend/src/screens/channels/Channels.tsx (hunk 1 above)
Comment (rolznz) [reply frontend/src/screens/channels/Channels.tsx:70]: true, thanks!

**Review by rolznz — APPROVED** [review]
utACK

**Review by bumi — COMMENTED** [review]
utACK

**Pull request merged**